api.c

```diff
@@ -572,10 +571,6 @@ int tcmu_emulate_evpd_inquiry(
 		/* Optimal xfer length */
 		memcpy(&data[12], &val32, 4);
 
-		/* MAXIMUM WRITE SAME LENGTH */
-		val64 = htobe64(VPD_MAX_WRITE_SAME_LENGTH);
-		memcpy(&data[36], &val64, 8);
-
```

Comment:
This does not seem to work for linux. Zero means no support or use lowest possible safe value it knows about which is 0xffff for both WS10 and WS16. If we report the max we support for WS16 and it is larger than 0xffff, it will just figure things out and do the right thing.

I wonder if the linux behavior is due to buggy targets like us that would report 0x3c for the b0 vpd page, but not actually support WS and so the MAX WS was zero in there.